pages/mesosphere/dcos/1.14/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -100,6 +100,8 @@ This page contains the configuration parameters for both DC/OS Enterprise and DC
 | [ca_certificate_path](#ca-certificate-path-enterprise)                   | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/mesosphere/dcos/1.14/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | [ca_certificate_key_path](#ca-certificate-key-path-enterprise)           | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/mesosphere/dcos/1.14/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | [ca_certificate_chain_path](#ca-certificate-chain-path-enterprise)       | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/mesosphere/dcos/1.14/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
+| [exhibitor_tls_required](#exhibitor-tls-required-enterprise)             | When true, hard failures are raised if there are any failures during exhibitor TLS bootstrapping           |
```

Comment:
Let's capitalize Exhibitor everywhere in our docs patch... ideally with a search/replace :-).